package.json

```diff
@@ -27,8 +28,12 @@
     "webpack-dev-server": "^1.10.1"
   },
   "scripts": {
+    "browse": "open http://localhost:5001",
     "build": "webpack",
     "clean": "rm -fr tmp/coverage/javascript/* && rm -fr dist/*",
+    "dev-server": "node webpack-dev-server.js",
+    "start": "http-server -p 5001",
+    "lint": "eslint --ignore-pattern dist ./; exit 0",
```

Comment:
More background on the `exit 0` hack: https://github.com/eslint/eslint/issues/2409#issuecomment-103768546